.github/workflows/pr-lint-monorepo.yml

```diff
@@ -1,37 +1,39 @@
 name: Run lint checks potentially affecting projects across the monorepo
 on:
-  pull_request:
-    branches:
-      - 'trunk'
+    pull_request:
+        paths-ignore:
+            - '**/changelog/**'
```

Comment:
This check includes changelog validation, so should not ignore this path

Reply:
Good point. Will fix.

Reply:
fixed https://github.com/woocommerce/woocommerce/pull/36280/commits/06e8bec6541b1e4f63e28b8089900c2dd1b2d699